mod/profiles.php

```diff
@@ -535,18 +535,18 @@ function profiles_content(App $a) {
 		$personal_account = !(in_array($a->user["page-flags"],
 					[User::PAGE_FLAGS_COMMUNITY, User::PAGE_FLAGS_PRVGROUP]));
 
-		$detailled_profile = (PConfig::get(local_user(), 'system', 'detailled_profile') AND $personal_account);
+		$detailed_profile = (PConfig::get(local_user(), 'system', 'detailled_profile') AND $personal_account);
```

Comment:
There is another detai**ll**ed_profile.

Reply:
Yes but it would reset all existing user settings to the default value, and I didn't want to write a post update script.

Reply:
Ah! I understand.